Mathlib/AlgebraicGeometry/EllipticCurve/Weierstrass.lean

```diff
@@ -181,6 +181,54 @@ lemma c_relation : 1728 * W.Δ = W.c₄ ^ 3 - W.c₆ ^ 2 := by
 
 end Quantity
 
+section ModelsWithJInvariant_0_1728
+
+variable (R : Type u) [CommRing R]
+
+/-- The Weierstrass curve $Y^2 + Y = X^3$. -/
+def ofJInvariant0 : WeierstrassCurve R :=
+  ⟨0, 0, 1, 0, 0⟩
+
+lemma ofJInvariant0_c₄ : (ofJInvariant0 R).c₄ = 0 := by
+  simp only [ofJInvariant0, c₄, b₂, b₄]
+  ring1
```

Comment:
These can probably be `rw`'s and `norm_num1`'s (same with the three other instances below).
```suggestion
lemma ofJInvariant0_c₄ : (ofJInvariant0 R).c₄ = 0 := by
  rw [ofJInvariant0, c₄, b₂, b₄]
  norm_num1
```